File: CS/SeriesItemTemplateSample/Model/GdpSeries.cs
using System;
using System.Collections.Generic;

namespace SeriesItemTemplateSample.Model {
    #region #Model
    public class GdpSeries {
        public String CountryName { get; set; }
        public IEnumerable<Gdp> Values { get; set; }
    }

    public class Gdp {
        public int Year { get; set; }
        public double Value { get; set; }
    }
    #endregion #Model
}

File: VB/SeriesItemTemplateSample/Model/GdpSeries.vb
Imports System.Collections.Generic

Namespace SeriesItemTemplateSample.Model

#Region "#Model"
    Public Class GdpSeries

        Public Property CountryName As String

        Public Property Values As IEnumerable(Of Gdp)
    End Class

    Public Class Gdp

        Public Property Year As Integer

        Public Property Value As Double
    End Class
#End Region  ' #Model
End Namespace
